subdir/attestationd/src/spdmlite.cpp

```diff
@@ -100,11 +101,39 @@ net::awaitable<void> onNeighbhorFound(
     sdbusplus::asio::object_server& dbusServer, SpdmHandler& spdmHandler,
     SpdmResponderIface& spdmResponder,
     std::shared_ptr<SpdmDeviceIface>& spdmDevice, const std::string& remotePort,
-    const boost::system::error_code& ec, const std::string& propVal)
+    const boost::system::error_code& ec, std::optional<sdbusplus::message_t> m)
 {
-    LOG_INFO("Neighbour LLDP ManagementAddressIPv4 changed: {}", propVal);
-    SpdmDeviceIface::ResponderInfo responderInfo{"device1", propVal,
-                                                 remotePort};
+    if (!m)
```

Comment:
you can also check actually D-bus error if you want to look different D-bus errors
sd_bus_error* dbusError = *m.get_error();
        if (dbusError != nullptr)
        {
        }